### ISSUE TITLE
Increasing amount of useable flash storage with Arduino Mega

### ISSUE THREAD

**Issue by ghost** [zrafa/30pin-simm-ram-arduino#1]
As the title suggest, would using a Mega, and interfacing with the remaining pins of the RAM module, allow the rest of the storage to be accessed?

**Comment by zrafa** [comment]
Yes, you are right. But, of course, some extra lines in source code is needed as well. I do not remember where now. If you want to try just tell me and we can modify the source code.
